docs/django/first-steps-with-django.rst

```diff
@@ -153,6 +153,57 @@ concrete app instance:
     You can find the full source code for the Django example project at:
     https://github.com/celery/celery/tree/main/examples/django/
 
+Trigger tasks at the end of the database transaction
```

Comment:
Should I mention a `.. versionadded::` in this part?

Reply:
I think that would help

Reply:
As it's more of a new feature than a bugfix, I'm assuming that it'll be part of 5.4 rather than the next 5.3.x

Reply:
yes 5.4 it should be